### DISCUSSION TITLE
Do I Have to Intstall Spotube In SysWOW64?

### DISCUSSION THREAD

**Discussion by Bj007pro** [KRTirtho/spotube#525] [Q&A]
I am using the installer named: _Spotube-windows-x86_64-setup.exe_

**Comment by KRTirtho** [comment]
No it shouldn't. It should be in `C:/Program Files/Spotube`. Something has messed up the installer metadata. Maybe a corrupted download?

**Reply by Bj007pro** [reply]
> No it shouldn't. It should be in `C:/Program Files/Spotube`. Something has messed up the installer metadata. Maybe a corrupted download?

Still the same thing even after redownloading it from the [website](https://spotube.netlify.app/).

**Reply by KRTirtho** [reply]
Can you change the path from the installer?

**Reply by Bj007pro** [reply]
Well, yeah. But if I'm downloading an installer file that is doing something that it shouldn't I prefer to not use it to be safe.

**Reply by KRTirtho** [reply]
Yes of course it's your choice whether you run or not run the installer
But please do a quick virus scan, to make sure nothing wrong is in your device
This usually doesn't happen. Something's messing with the installer metadata

**Reply by KRTirtho** [reply]
Btw, you can use th Microsoft's official Winget package mansger to install spotube
Or use the chocolatey package manager
Spotube is also available in scoop package manager

**Reply by Bj007pro** [reply]
Tonight I thought I would try to use the latest nightly build version, and it's still the same issue: 

![image](https://github.com/KRTirtho/spotube/assets/24400879/96bcb2e5-96f6-4a94-b63a-ffb953686117)

> Something has messed up the installer metadata. Maybe a corrupted download?

You say that there is something on my PC that is messing up with the installer's metadata, but this doesn't happen with any other program I've recently installed before or since I've started this issue ticket.

**Reply by KRTirtho** [reply]
You can change the destination folder from here

**Reply by Bj007pro** [reply]
As, I said, previously;
> Well, yeah. But if I'm downloading an installer file that is doing something that it shouldn't I prefer to not use it to be safe.

**Reply by KRTirtho** [reply]
Yes and I also said
> Yes of course it's your choice whether you run or not run the installer
